sonar-python-plugin/src/main/java/org/sonar/plugins/python/pylint/PylintConfiguration.java

```diff
@@ -22,10 +22,14 @@
 import java.io.File;
 import org.apache.commons.lang.StringUtils;
 import org.sonar.api.BatchExtension;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonar.api.BatchExtension'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-235b&issues=AWDqhpSPo_TTbi26M1MA&open=AWDqhpSPo_TTbi26M1MA)

[ISSUE_KEY]: AWDqhpSPo_TTbi26M1MA